Doc/library/ipaddress.rst

```diff
@@ -727,6 +751,11 @@ dictionaries.
       Refer to the corresponding attribute documentation in
       :class:`IPv4Network`.
 
+   .. method:: next_network(next_prefix=None)
```

Comment:
This should be moved with the other methods on line 749

Reply:
Thanks for your review.  Sure, I can move this, however I believe it is already near the methods as above it is the `compare_networks` method and below is an attribute.  Is this not correct? Or have I misunderstood?

Reply:
I meant that you don’t have to copy the two lines below if you move the method line in the same block as all the others above.  No reason to single out this one method!

Reply:
Oh I see, ok good point.  I have updated it now.  Out of interest, who should resolve the Github comments after a fix has been made? The person making the comment (to verify the fix?) or the person who has fixed it? I have left them as unresolved although I think I have fixed everything (so far)!

Reply:
Sometimes the changes close the comments, sometimes the PR author resolve them, but if you reply to the comment and resolve then the reply gets hidden.